_studio/mfx_lib/encode_hw/mjpeg/src/mfx_mjpeg_encode_vaapi.cpp

```diff
@@ -187,25 +188,39 @@ mfxStatus VAAPIEncoder::CreateAccelerationService(mfxVideoParam const & par)
     }
 
     // Configuration
-    VAConfigAttrib attrib;
+    mfxPriority priority;
+    VAConfigAttrib attrib[2];
 
-    attrib.type = VAConfigAttribRTFormat;
+    attrib[0].type = VAConfigAttribRTFormat;
+    attrib[1].type = VAConfigAttribContextPriority;
 
     vaSts = vaGetConfigAttributes(m_vaDisplay,
                           VAProfileJPEGBaseline,
                           VAEntrypointEncPicture,
-                          &attrib, 1);
+                          &attrib[0], 2);
```

Comment:
see my comment about ```vector``` below

Reply:
I am lazy to dig for @mgonchar comment, but suspect I will write the same...

imho, hardcoded 2 is avoidable. Use std::vector attrib(2), use attrib.size()

Reply:
Done